docs/en/cloud/AWS-EMR.md

```diff
@@ -71,49 +71,40 @@ $ aws emr create-default-roles
 The `create-cluster` command requires passing in multiple flags to successfully execute:
 - `release-label`: The version of EMR to install with.
 The current version of Alluxio is compatible with `emr-5.25.0`.
-- `custom-ami-id`: The Alluxio Marketplace Enterprise Edition AMI ID.
-To find the latest AMI ID for a particular region, run the following AWS CLI command:
-```console
-aws ec2 describe-images \
---owners 'aws-marketplace' \
---filters 'Name=product-code,Values=6axlevynmkcnoq7ximzar80bx' \
---query 'sort_by(Images, &CreationDate)[-1].[ImageId]' \
---output text \
---region us-east-1
-```
-where the value for `--region` can be set to the desired region.
-
 - `instance-count`: The number of nodes to provision for the cluster.
 - `instance-type`: The instance type to provision with.
 Make sure you pick an instance type supported by the Alluxio marketplace AMI.
-Note that your account can be limited in the number of instances you can launch; check your instance limits [here](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Limits:).
+Note that your account can be limited in the number of instances you can launch;
+check your instance limits [here](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Limits:).
```

Comment:
This link is specifically to us-east-1. I'm assuming that it's required for a region to be specified - but it could be misleading to someone who normally operates out of, say, us-west-1.